docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,32 @@
 title: Changelog
 ---
 
+## 13.0.0
+
+_Released 08/29/2023_
```

Comment:
we need to sync the changelog again once v13 in cypress-api is in `develop`, and the items itended for `12.17.5` are now in `13.0.0` and will need to be added here